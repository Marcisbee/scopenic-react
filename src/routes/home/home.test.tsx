import React from 'react';
import { shallow } from 'enzyme';
import Home from './home';

describe('home', () => {
  it('renders without crashing', () => {
    const component = shallow(<Home />);
    expect(component).toMatchSnapshot();
  });
});
