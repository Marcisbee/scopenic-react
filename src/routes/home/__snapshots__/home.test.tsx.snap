// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`home renders without crashing 1`] = `
<div>
  <h1>
    Home
  </h1>
</div>
`;
